### DISCUSSION TITLE
version of manjaro with fyne

### DISCUSSION THREAD

**Discussion by ghost** [fyne-io/fyne#3284] [Q&A] [answered]
Hi all.

I would like to know if you all thought of developing a version of manjaro with fyne?

**Note:** I'm thinking of developing a version of manjaro with fyne and would like to know everyone's opinion on this idea.

**Comment by Jacalz** [comment, marked as answer]
I am not entirely sure that I understand what you mean. Are you suggesting having https://github.com/fyne-io/fynedesk as a desktop on Manjaro or are are you trying to decide on using it for developing Fyne applications? In either way it should not be a problem. There are installation steps for Arch Linux: https://developer.fyne.io/started/.

**Reply by ghost** [reply]
Hi Jacalz.

> Are you suggesting having https://github.com/fyne-io/fynedesk as a desktop on Manjaro?

yeah.

**Reply by Jacalz** [reply]
Using it on Manjaro should just be a case of compiling it and installing it alongside your other desktop environment. If you want to create a full spin with only FyneDesk, it should certainly be possible but does of course require a bit more work

**Reply by andydotxyz** [reply]
Sounds fun for sure. Probably could use a lot of the repos we have at https://github.com/fyshos/ as well.
That project is based on Debian, but I would imagine there is a lot of overlap.
The biggest challenge is getting a custom bootable .ISO but maybe manjaro has this covered?
I would recommend https://github.com/fyshos/fin as the login manager :).
